core/src/components/fab-button/fab-button.scss

```
@@ -45,6 +45,7 @@
   --margin-bottom: calc((#{$fab-size} - var(--height)) / 2);
   --transition: background-color, opacity 100ms linear;
   --ripple-color: currentColor;
+  --border-radius: #{$fab-border-radius};
 
```

```suggestion
  --border-width: 0;
  --border-style: none;
  --border-color: initial;
```

yep @brandyscarney! i think so, otherwise that CSS vars applied to any parent... a ion-modal for example, would affect all ion-fab-button inside it